### PR TITLE
Add auth commands

### PR DESCRIPTION
**Pull request by nmarcopo**
Adds `auth` commands to authenticate the chronoctl session.

### CODE REVIEW

File: src/cmd/pkg/auth/auth.go [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+	return store, nil
+}
+
+type loginOpts struct {
+	orgName           string
+	loginPath         string
+	skipSetDefaultOrg bool
+}
+
+func (o *loginOpts) registerFlags(cmd *cobra.Command) {
```
Comment (bg451) [inline src/cmd/pkg/auth/auth.go:94]: nit: addFlags to match other opts

File: src/cmd/pkg/auth/auth.go [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+				return errors.WithStack(errMustIncludeOrgName)
+			}
+			return setDefaultOrg(store, args[0])
+		},
+	}
+	return cmd
+}
+
+func (c *subcommand) newPrintAccessTokenCmd() *cobra.Command {
+	cmd := &cobra.Command{
```
Comment (bg451) [inline src/cmd/pkg/auth/auth.go:148]: Consider using https://pkg.go.dev/github.com/spf13/cobra#ExactArgs for arg length validation as it'll print out the usage.

File: src/cmd/pkg/auth/auth.go [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+		},
+	}
+
+	opts.registerFlags(cmd)
+	return cmd
+}
+
+func (c *subcommand) newSetDefaultOrgCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "set-default-org ORG_NAME",
```
Comment (bg451) [inline src/cmd/pkg/auth/auth.go:130]: ditto the ExactArgs comment.

**Review by bg451 — APPROVED** [review]

File: src/cmd/pkg/auth/auth.go (hunk 1 above)
Comment (nmarcopo) [reply src/cmd/pkg/auth/auth.go]: SGTM - [`3975d5c` (#27)](https://github.com/chronosphereio/chronoctl-core/pull/27/commits/3975d5c62f75b1f6382a4c154e63a518d622b7c6)

File: src/cmd/pkg/auth/auth.go (hunk 2 above)
Comment (nmarcopo) [reply src/cmd/pkg/auth/auth.go:146]: Thanks for the pointer! [`1d1a4cd` (#27)](https://github.com/chronosphereio/chronoctl-core/pull/27/commits/1d1a4cd2f4a7650d59c87dcc4336a2064336deff)

File: src/cmd/pkg/auth/auth.go (hunk 3 above)
Comment (nmarcopo) [reply src/cmd/pkg/auth/auth.go:130]: [`1d1a4cd` (#27)](https://github.com/chronosphereio/chronoctl-core/pull/27/commits/1d1a4cd2f4a7650d59c87dcc4336a2064336deff)

**Pull request merged**